libs/auth/scripts/generate_dev_keys_and_certs.ts

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable no-console */
 import { Buffer } from 'buffer';
-import * as fs from 'fs/promises';
+import { promises as fs } from 'fs';
```

Comment:
Another change to avoid Cypress errors, this time `Can't resolve 'fs/promises'`, which actually seems to be a Webpack error (Cypress uses Webpack as its default preprocessor).

This import change may no longer be necessary since I'm importing `@votingworks/auth/src/mock_file_card` instead of all of `@votingworks/auth` (and `@votingworks/auth/src/mock_file_card` uses standard `fs` instead of `fs/promises`), but I kept it to avoid headaches down the road.

I'm sure we could (and probably should) update Cypress to use Vite as its preprocessor, but that was out of scope for this PR.